teuthology/task/install.py

```diff
@@ -65,7 +65,10 @@
     'python-ceph',
     'rbd-fuse',
 ]
-
+RHVERSION={}
+RHVERSION['ceph']={
+    '1.3.0':1    
+}
```

Comment:
@andrewschoen asked a while ago:
"What's the purpose of this? Are you just trying to validate that the version put in rhbuild is valid?

If so, maybe just a list of version would be better?"